tests/acceptance/features/apiSharingNg/linkShare.feature

```diff
@@ -950,3 +888,745 @@ Feature: Create a share link for a resource
       | 123             |
       | password        |
       | ownCloud        |
+
+  @issue-7879
+  Scenario Outline: create a share link of a folder inside project-space
+    Given using spaces DAV path
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "projectSpace" with the default quota using the Graph API
+    And user "Alice" has created a folder "folderToShare" in space "projectSpace"
+    When user "Alice" creates the following link share using the Graph API:
+      | resource        | folderToShare      |
+      | space           | projectSpace       |
+      | permissionsRole | <permissions-role> |
+      | password        | %public%           |
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+      """
+      {
+        "type": "object",
+        "required": [
+          "hasPassword",
+          "id",
+          "link"
+        ],
+        "properties": {
+          "hasPassword": {
+            "const": true
+          },
+          "id": {
+            "type": "string",
+            "pattern": "^[a-zA-Z]{15}$"
+          },
+          "link": {
+            "type": "object",
+            "required": [
+              "@libre.graph.displayName",
+              "@libre.graph.quickLink",
+              "preventsDownload",
+              "type",
+              "webUrl"
+            ],
+            "properties": {
+              "@libre.graph.displayName": {
+                "const": ""
+              },
+              "@libre.graph.quickLink": {
+                "const": false
+              },
+              "preventsDownload": {
+                "const": false
+              },
+              "type": {
+                "const": "<permissions-role>"
+              },
+              "webUrl": {
+                "type": "string",
+                "pattern": "^%base_url%/s/[a-zA-Z]{15}$"
+              }
+            }
+          }
+        }
+      }
+      """
+    Examples:
+      | permissions-role |
+      | view             |
+      | edit             |
+      | upload           |
+      | createOnly       |
+      | blocksDownload   |
+
+
+  Scenario: create an internal link share of a folder inside project-space
```

Comment:
could you please expand this scenario for this https://github.com/owncloud/ocis/issues/8960

Reply:
will be done in https://github.com/owncloud/ocis/issues/9046